LiskHQ/lisk-sdk
# Add semgrep config and fix identified issues

### What was the problem?

This PR resolves #8474 and resolves #8472

### How was it solved?

- Add semgrep config

### How was it tested?

- Check the issues 
- run semgrep `semgrep scan --config semgrep.yml`

### Additional Information
- semgrep is not added to the CI for now. We may consider adding it to CI if we find the tool useful
- semgrep can be installed with https://semgrep.dev/docs/getting-started/
